### DISCUSSION TITLE
Problemi con "condizioni di attivazione" e lingua

### DISCUSSION THREAD

**Discussion by LucaPero12** [globaleaks/globaleaks-whistleblowing-software#3871] [Q&A] [answered]
Ciao, sono nuovo di Globaleaks, mi piace perchè molto semplice, ma ci sono delle funzioni che non vanno/non caipsco come farle andare:
- Condizioni/trigger: voglio che una domanda compaia solo nel caso si risponda "si a quella precedente". Ho impostato le domande nel questionario, ma quando faccio "+Aggiungi" alle "condizioni di attivazione" della domanda soggetta al trigger, il tab di selezione compare vuoto! Mentre mi aspetterei di poter selezionare la domanda (e la risposta) trigger. Sapete come mai?
- non capisco proprio come impostare più lingue per lo stesso questionario, mi potete aiutare?
Scaricando il json del questionario di prova (che ha sia trigger che più lingue che funzionano correttamente) ho visto i campi che impostano le cose, quindi pensavo di poter ovviare scaricando il json del mio questionario, modificare i campi a mano e reimportarlo. Ma anche così mi da errore nell'importazione. Anzi pure esportandolo (cancellando quello attivo) e reimportandolo così com'è mi da errore di importazione! Potete aiutarmi?🫶🏻🙏🏻 Grazie

**Comment by evilaliv3** [comment, marked as answer]
- Il primo problema potrebbe essere un problema di refresh noto nel caso in cui tu abbia appena creato la domanda trigger.
- Relativamente alle lingue, per configurare una traduzione (e.g. l'inglese), opera in in quella lingua e salva. I dati e caricati e salvati operando in una lingua vengono salvati come traduzione in quella lingua.

**Reply by LucaPero12** [reply]
- Allora, ho fatto delle prove ed effettivamente con il refresh sembrava fosse andato (ma solo con le domande messe come modello, non quelle inserite direttamente nel questionario). Però ho notato che ora quando inserisco la condizione di attivazione la domanda sparisce correttamente dal questionario in visualizzazione, ma non compare più, neanche inserendo la risposta selezionata nella domanda trigger.😬 ....poi illuminarmi anche in questo?😅
- Invece per la questione della lingua grazie, facendo come dici ha funzionato.
Grazie mille
